src/isoCodeLists.js

```diff
@@ -217,3 +217,55 @@ export const depthDirections = {
   heightPositive: { en: "Depth Positive", fr: "Profondeur positive" },
   depthPositive: { en: "Height Positive", fr: "Hauteur positive" },
 };
+
+export const metadataScopeCodes = {
+  dataset: {
+    title: {
+      en: "dataset",
+      fr: "Ensembles de données"
+    },
+    text: {
```

Comment:
Did you mean to use these `text` fields somewhere? Could do a mouseover in the select box, or add to the question text

Reply:
good point, added a tooltip, thanks